avocado/utils/perf.py

```diff
@@ -0,0 +1,41 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; specifically version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Author: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>
+
+"""
+Linux perf tool related functions.
+"""
+
+from . import process
+
+
+class PerfError(Exception):
+    """
+    Base Exception Class for all exceptions
+    """
+
+
+def get_events(pattern):
```

Comment:
I'm skeptical of the value of this utility method, as it does exactly what something like this would:

```python
events = [line for line in process.run("perf list").stdout_text.splitlines() if pattern in line]
```

The only configurable aspect of this is the command itself.  So, IMO, this looks more like a astring/genio utility function then something that deserves its own `perf` module.

Reply:
@clebergnu thanks for review , we wrote this as perf utility as we have  more function that we are planing to write and push in avocado as maintenance point of that will be good  now it can be part of astring or `linux_utils` what going to introduce via #3091  later on future when we have more perf utility if needed  with help of you we can re categorizes